test/views/AccessibilityProblemConfirmationPageSpec.scala

```diff
@@ -43,6 +42,10 @@ class AccessibilityProblemConfirmationPageSpec
     val accessibilityProblemConfirmationPage = app.injector.instanceOf[AccessibilityProblemConfirmationPage]
     val content                              = accessibilityProblemConfirmationPage()
 
+    "pass accessibility checks" in {
+      content.toString() should passAccessibilityChecks
```

Comment:
future bit for scalatest-accessibility-linter I guess is to have another apply method that accepts HtmlAppendable and does delegates to `apply(html.toString)` https://github.com/hmrc/scalatest-accessibility-linter/blob/main/src/main/scala/uk/gov/hmrc/scalatestaccessibilitylinter/AccessibilityMatchers.scala#L31